tasks/main.yml

```diff
@@ -7,6 +7,16 @@
              src={{ nsq_download_dir }}/nsq.zip
              dest={{ nsq_download_dir }}
 
+- name: Install supervisor
```

Comment:
this should be part of a `install_RedHat.yml` - and split ubuntu / centos into 2 differnet setups.